recipes/pyhrf_package/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pyhrf" %}
+{% set version = "0.4.3" %}
+{% set sha256 = "23b406373dff7a8b49bf8eb674e2b9742b4409ab5b9ce430db7575c10119e515" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py3k or not np==111]
```

Comment:
@ainafp I made a mistake here and no package is getting build :unamused: 

Let me fix this... Hold on...